src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/SendEmailsForGoCardlessMandateEventsHandlerTest.java

```diff
@@ -48,43 +53,45 @@
             .withPayerFixture(payerFixture);
 
     private GoCardlessEventFixture goCardlessEventFixture = GoCardlessEventFixture.aGoCardlessEventFixture()
+            .withResourceType(GoCardlessResourceType.MANDATES)
             .withLinksOrganisation(organisationIdentifier);
 
     @Test
-    public void handle_onAFailedMandateGoCardlessEvent_shouldRegisterAPayEventAsMandateFailed() {
-        GoCardlessEvent goCardlessEvent = spy(goCardlessEventFixture.withAction("failed").toEntity());
+    public void assertEmailIsSentForFailedMandateGoCardlessEvent() {
+        GoCardlessEvent goCardlessEvent = goCardlessEventFixture.withAction("failed").toEntity();
 
         when(mockMandateQueryService.findByGoCardlessMandateIdAndOrganisationId(
                 goCardlessEvent.getLinksMandate().get(), goCardlessEventFixture.getLinksOrganisation()))
                 .thenReturn(mandateFixture.toEntity());
 
-        goCardlessMandateHandler.handle(goCardlessEvent);
+        sendEmailsForGoCardlessEventsHandler.handle(List.of(goCardlessEvent));
 
-        verify(userNotificationService).sendMandateFailedEmailFor(mandateFixture.toEntity());
+        verify(userNotificationService, times(1)).sendMandateFailedEmailFor(mandateFixture.toEntity());
```

Comment:
Don’t need `times(1)`: it’s the default.

Reply:
fixed